build.gradle

```diff
@@ -32,7 +32,5 @@ subprojects {
     apply plugin: 'com.palantir.baseline-eclipse'
     apply plugin: 'com.palantir.baseline-findbugs'
     apply plugin: 'com.palantir.baseline-idea'
-
-    sourceCompatibility = 1.7
```

Comment:
cool. dunno if we want to be explicit and set 1.8. i'm indifferent. removing may be the best.